apps/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 def isCI = System.getenv('CI') ? true : false
 
 def nodeSpec = {
-    version = '19.9.0'
+    version = '18.19.0'
```

Comment:
had to do this to get it compile locally.

Reply:
strange should be same as master , should not be part of this commit , master at 19.9.0 , maybe clean node_modules and rebuild.